qemu/tests/hugepage_mem_stress.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_test
+from virttest.utils_test import BackgroundTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu hugepage memory stress test.
+    Steps:
+    1) System setup hugepages on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Run memory heavy stress inside guest.
+    4) Check guest call trace in dmesg log.
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    def heavyload_install():
```

Comment:
let's call 

stress_app = virttest.utils_test.VMStress()

Reply:
sorry, regarding the windows heavyload tool, I don't know how virttest.utils_test.VMStress () calls it.

Reply:
like stress, pass arguments to it.

Reply:
ok changing.